docs/source/usage/deployment.rst

```diff
@@ -101,12 +123,15 @@ Deploy Your Framework
 
 This is where we take everything we've done up to this point and start the deployment.
```

Comment:
I changed the rest to toggle based on deployment options. It's just an idea, we don't have to do it this way. Maybe we should dedicate each deployment scenario their own .rst file and link to them in here instead.

Reply:
I love the toggle <3